.cloudbuild/image-ecr.yaml

```diff
@@ -0,0 +1,19 @@
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    secretEnv:
+      - STAGING_DOCS_ECR_KEY
+      - STAGING_DOCS_ECR_SECRET
+      - PRODUCTION_DOCS_ECR_KEY
+      - PRODUCTION_DOCS_ECR_SECRET
+    script: .cloudbuild/publish-image-ecr.sh 
+availableSecrets:
+  secretManager:
```

Comment:
Checking with the triggers in `ops`, and my understanding is that this should only ever be run on a merge to `main` - is that correct?

For context, I'm just checking that that this doesn't violate our "no secrets in PR builds" rule.

Reply:
That's correct. See https://github.com/gravitational/ops/pull/386/. The trigger should only work with push on main.

Reply:
That's correct. See gravitational/ops#386. The trigger should only work with push on main.